src/Microsoft.SqlTools.ServiceLayer/Microsoft.SqlTools.ServiceLayer.csproj

```diff
@@ -39,10 +39,6 @@
 			</PropertyGroup>
 			<ItemGroup>
 				<PackageReference Include="Microsoft.SqlServer.Migration.Assessment" />
-				<Content Include="./Migration/Metadata/**">
-					<Link>%(RecursiveDir)%(Filename)%(Extension)</Link>
-					<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-				</Content>
 				<Compile Include="**/*.cs" Exclude="**/obj/**/*.cs" />
```

Comment:
+@corivera,

especially concerning this csproj -- all I removed was the Content Include tag, but not sure if anything else was needed? 

(Once we get the NuGet to be arm64 compatible maybe we can revisit the workaround you added here -- but for now just curious about the current way the package is being consumed)

Reply:
There's another reference to the migration metadata folder in the integration tests csproj that could be removed: https://github.com/microsoft/sqltoolsservice/blob/main/test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/Microsoft.SqlTools.ServiceLayer.IntegrationTests.csproj#L27 

You can probably remove that whole Otherwise block, since it only deals with copying the metadata files.

Reply:
Got it, thanks! I'll have a look at that one too. When you say "remove the whole Otherwise block", do you mean the one in the IntegrationTests csproj specifically? Or can I do it in this one too? I notice there's the `<Compile Include="**/*.cs" Exclude="**/obj/**/*.cs" />` line right below my changes, which I didn't want to touch

Reply:
Only the IntegrationTests one. The compile elements exclude the migration source files in the dotnet tool build, since it's not supported on arm64 yet.

Reply:
@corivera Cool, thanks. Pushed a new iteration to remove the reference from IntegrationTests